dateutil/tz/tz.py

```diff
@@ -13,13 +13,8 @@
 import sys
 import os
 import bisect
-import copy
 
-from operator import itemgetter
-
-from contextlib import contextmanager
```

Comment:
I'm pretty sure these are leftovers from things moved out of this module, good to go.